saleor/saleor-dashboard
# Allow refreshing proper token based on token owner

I want to merge this change because it fixes the infinite loading of the dashboard if the end-user logs in with the cloud auth plugin and then wants to login with an email/password afterward.

SDK branch: https://github.com/saleor/saleor-sdk/pull/196


### How to test it?
1. Login with email/password
2. Set `_saleorAuthPluginId` with the value `cloud_auth.CloudAuthorizationPlugin` inside dashboard local storage
3. Reload the page
4. Dashboard should load normally



**PR intended to be tested with API branch:** 

### Screenshots



### Pull Request Checklist



1. [ ] This code contains UI changes
5. [ ] All visible strings are translated with proper context including data-formatting
6. [ ] Attributes `[data-test-id]` are added for new elements
7. [ ] Changes are mentioned in the changelog
8. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config



API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
APPS_MARKETPLACE_API_URI=https://apps.staging.saleor.io/api/v2/saleor-apps

### Do you want to run more stable tests?

To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] giftCard
3. [ ] category
5. [ ] collection
6. [ ] attribute
9. [ ] productType
10. [ ] shipping
11. [ ] customer
12. [ ] permissions
13. [ ] menuNavigation
14. [ ] pages
15. [ ] sales
16. [ ] vouchers
17. [ ] homePage
18. [ ] login
19. [ ] orders
20. [ ] products
21. [ ] app
22. [ ] plugins
23. [ ] translations
24. [ ] navigation
25. [ ] variants
26. [ ] payments

CONTAINERS=1
